src/Modules/OrchardCore.Commerce/Events/WorkflowShoppingCartEvents.cs

```diff
@@ -91,15 +90,42 @@ private T GetOutput<T>(IEnumerable<WorkflowExecutionContext> contexts, string ou
             .SelectWhere(context => context.Output.GetMaybe(outputName))
             .FirstOrDefault();
 
-        return output switch
+        if (output is string text && typeof(T) == typeof(SerializableLocalizedHtmlString))
```

Comment:
can this be changed to `output is SerializableLocalizedHtmlString t` then return t?